tensorflow_lite_support/java/src/java/org/tensorflow/lite/support/image/BoundingBoxUtil.java

```diff
@@ -197,6 +197,43 @@ private static RectF convertFromBoundaries(
     }
   }
 
+  private static RectF convertFromUpperLeft(
+      float[] values, CoordinateType coordinateType, int height, int width) {
+    if (coordinateType == CoordinateType.PIXEL) {
+      float left = values[0];
+      float top = values[1];
+      float w = values[2];
+      float h = values[3];
+
+      return new RectF(
+        left, top, left + w, top + h);
+    } else {
+      return new RectF(
+        values[0] * width, values[1] * height, width * values[2], height * values[3]);
```

Comment:
Nit: Let's keep the expressions in the same order, like foo * value[bar].

Reply:
Correct expressions seem to be

values[0] * imageWidth
values[1] * imageHeight
(values[0] + values[2]) * imageWidth
(values[1] + values[3]) * imageHeight